.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
I don't see a reference to this in the other changed files. I realize this is probably part of your boilerplate, but might consider removing since it does not add anything to your project.

Reply:
This is just a .env file to prevent create-react-app's preboot checks from preventing launch. There's some conflict with my device's global webpack install, but its only like 3 versions off.